src/tags/kick.js

```diff
@@ -56,7 +56,12 @@ module.exports =
                     return error(`You don't have permission to kick ${user.username}!`);
                 default: //Error occurred
                     throw state;
-            }
+            }*/
+    
+            if (typeof response[1] == 'string' && response[1].startsWith('`'))
+                return Builder.util.error(subtag, context, response[1]);
+    
+            return response[1];
```

Comment:
Check and throw if there is an error
```suggestion
            if (typeof response[1] != 'string')
                throw response[1];
            if (response[1].startsWith('`'))
                return Builder.util.error(subtag, context, response[1].slice(1, -1));
    
            return response[1];
```